packages/data/src/ethers.ts

```diff
@@ -40,16 +45,16 @@ export default class SemaphoreEthers {
      * @param options Configuration options for the ethers provider and the Semaphore contract.
      */
     constructor(networkOrEthereumURL: EthersNetwork | string = defaultNetwork, options: EthersOptions = {}) {
-        checkParameter(networkOrEthereumURL, "networkOrSubgraphURL", "string")
+        requireString(networkOrEthereumURL, "networkOrSubgraphURL");
```

Comment:
`;` is not allowed in this repo. Please, run `yarn format:write` to format the code automatically.